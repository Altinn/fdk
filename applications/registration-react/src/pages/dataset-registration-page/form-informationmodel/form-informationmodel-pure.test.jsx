import React from 'react';
import { shallow } from 'enzyme';
import {
  FormInformationModelPure,
  renderInformationModel
} from './form-informationmodel-pure.component';

let defaultProps;
let wrapper;

beforeEach(() => {
  defaultProps = {};
  wrapper = shallow(<FormInformationModelPure {...defaultProps} />);
});

test('should render FormInformationModelPure, { renderS correctly', () => {
  expect(wrapper).toMatchSnapshot();
});

test('should render renderInformationModel correctly', () => {
  wrapper = shallow(renderInformationModel(defaultProps));
  expect(wrapper).toMatchSnapshot();
});
