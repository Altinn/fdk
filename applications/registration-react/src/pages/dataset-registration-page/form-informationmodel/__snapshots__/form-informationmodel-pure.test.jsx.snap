// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FormInformationModelPure, { renderS correctly 1`] = `
<form>
  <div
    className="form-group"
  >
    <div
      className="mt-4"
    >
      <div
        className="form-group"
      >
        <withStateHandlers(Helptext)
          term="Dataset_informationModel"
          title="Informasjonsmodell"
        />
        <FieldArray
          component={[Function]}
          linkLabel="Lenke"
          name="informationModel"
          titleLabel="Tittel"
        />
      </div>
    </div>
  </div>
</form>
`;

exports[`should render renderInformationModel correctly 1`] = `<div />`;
